src/modules/m_spanningtree/treeserver.cpp

```diff
@@ -37,6 +37,7 @@ TreeServer::TreeServer()
 	, Parent(NULL), Route(NULL)
 	, VersionString(ServerInstance->GetVersionString())
 	, fullversion(ServerInstance->GetVersionString(true))
+	, rawversion(INSPIRCD_VERSION)
```

Comment:
This will make servers which didn't send this show the local version. This seems like it would be undesirable behaviour. Maybe default to something else instead?